dts/arc/tenstorrent/tt_blackhole_fixed_partitions.dtsi

```diff
@@ -6,68 +6,102 @@
 
 / {
 	partitions {
-		compatible = "fixed-partitions";
+		compatible = "tenstorrent,tt-boot-fs", "fixed-partitions";
 		#address-cells = <1>;
 		#size-cells = <1>;
 
+		flash-device-size = <0x1000000>;
+		flash-block-size = <0x1000>;
+
 		cmfwcfg@14000 {
 			label = "cmfwcfg";
-			reg = <0x14000 0x38>; /* 81920, 56 bytes */
+			reg = <0x14000 0x1000>;
+			binary-path = "$BUILD_DIR/smc/generated_board_cfg/$BOARD_REV/fw_table.bin";
+			read-only;
+		};
+
+		origcfg@15000 {
+			label = "origcfg";
+			reg = <0x15000 0x1000>;
+			binary-path = "$BUILD_DIR/smc/generated_board_cfg/$BOARD_REV/fw_table.bin";
+			read-only;
 		};
 
-		cmfw@15000 {
+		cmfw@10000000 {
 			label = "cmfw";
-			reg = <0x15000 0x15248>; /* 86016, 86600 bytes */
+			reg = <0x10000000 0x1a000>;
```

Comment:
Might be worth using `DT_SIZE_K` or `DT_SIZE_M` macros here for readability

Reply:
The devicetree parser edtlib doesn't support #define or #include in the devicetree, so I'm not sure how to use those macros